cmd/parallel-manager.go

```diff
@@ -252,11 +252,7 @@ func availableMemory() (available uint64) {
 
 	} // for all other platforms limits are based on virtual memory.
 
-	memStats, err := mem.VirtualMemory()
-	if err != nil {
-		return
```

Comment:
maybe it is better to keep this to return 8Gb instead of 0Gb as available memory, 8GB is better than zero.

Reply:
8gb is too high right? @vadmeste I will perhaps fit this into 4Gb